### PR TITLE
Some small improvements to the SQL Module

### PR DESCRIPTION
**Pull request by adamretter**
1. You can now get results back from calling Stored Procedures
2. You can now specify `<sql:null/>` inside a `<sql:param>` when calling a stored procedure, see https://github.com/eXist-db/exist/issues/2139

### CODE REVIEW

File: extensions/modules/sql/pom.xml [hunk 1]
```diff
@@ -37,6 +37,11 @@
             <artifactId>xml-apis</artifactId>
         </dependency>
 
+        <dependency>
```
[the remark this answers is not in the document]
Comment (adamretter) [reply extensions/modules/sql/pom.xml]: No the scope should be compile

[the remark this answers is not in the document]
Comment (dizzzz) [reply extensions/modules/sql/pom.xml]: provided then?

[the remark this answers is not in the document]
Comment (adamretter) [reply extensions/modules/sql/pom.xml]: I think it's fine as is. Do you see something wrong?

**Comment by PieterLamers** [comment]
Thanks guys, that's great news. I hope  I will be able to try it out soon.

**Pull request merged**